index.html

```diff
@@ -1,53 +1,276 @@
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+      </section>
+      <!-- end of work section-->
+
+      <section class="about">
+        <h2 class="headline_title">About Myself</h2>
+        <p class="about_details">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h3 class="social_title blue">LET’S CONNECT</h3>
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/twitter.svg"
+                alt="Twitter"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/linkedin-icon.svg"
+                alt="Linkedin"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/m.svg" alt="M"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/github.svg" alt="Github"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img class="social_icon" src="./images/doit.svg" alt="Diot"
+            /></a>
+          </li>
+        </ul>
+        <div class="action">
+          <button type="button" class="default_btn">Get my resume</button>
+        </div>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Languages
+            <img
+              class="down_arrow_icon"
+              src="./images/down-arrow-icon.png"
+              alt="down"
+            />
+          </h3>
+          <li class="ul_item">
+            <img src="./images/js-icon.png" alt="Javascript" /><span
+              >Javascript</span
+            >
+          </li>
+          <li class="ul_item">
+            <img src="./images/html-icon.png" alt="HTML" /><span>HTML</span>
+          </li>
+          <li class="ul_item">
+            <img src="./images/css-icon.png" alt="CSS" /><span>CSS</span>
+          </li>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Frameworks
+            <img
+              class="arrow_icon"
+              src="./images/right-arrow-icon.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
+        <ul class="about_ul">
+          <h3 class="sub_title">
+            Skills
+            <img
+              class="arrow_icon"
+              src="./images/right-arrow-icon.png"
+              alt="down"
+            />
+          </h3>
+        </ul>
```

Comment:
- [ ] In this piece of code, we have a list of categories("languages", "frameworks" and "skills) and one other list of languages inside one item of the parent list, so a better approach would be to use one `ul` to group all the categories and each category should be inside an `li` tag, and inside the "languages" `li` tag there should be the 'language' card and another `ul` with all the other languages in `li` tags. Please refactor the code so it can be semantically correct.

Reply:
@leonardodiasb  that was the same i did

<ul class="social_links">
          <li class="social_link">
            <a href="#"
              ><img
                class="social_icon"
                src="./images/twitter.svg"
                alt="Twitter"
            /></a>
          </li>

Reply:
`<ul class="social_links">
          <li class="social_link">
            <a href="#"
              ><img
                class="social_icon"
                src="./images/twitter.svg"
                alt="Twitter"
            /></a>
          </li>`